dvc/utils/fs.py

```diff
@@ -95,15 +95,19 @@ def move(src, dst, mode=None):
     dst = os.path.abspath(dst)
     tmp = f"{dst}.{uuid()}"
 
+    try:
+        if mode is not None:
+            os.chmod(src, mode)
+    except OSError:
+        # File not owned by us, raise exception
```

Comment:
The exception string seems to pass the same information. Do we need the comment?

Reply:
Also, since we are only raising another exception, we can make use of `reraise`, an example is [here](https://github.com/iterative/dvc/blob/411b38896014ae004d0782e25dbd49f5d33af2b7/dvc/external_repo.py#L170)

Reply:
1. Good point. 
2. Not sure what you meant so I introduced a new exception and used `reraise`.